odl/contrib/fom/test/test_supervised.py

```diff
@@ -6,24 +6,58 @@
 # v. 2.0. If a copy of the MPL was not distributed with this file, You can
 # obtain one at https://mozilla.org/MPL/2.0/.
 
-"""Tests for supervised FoMs."""
+"""Test for the figures of merit (FOMs) that use a known ground truth."""
 
 from __future__ import division
 import numpy as np
 import pytest
 import scipy.signal
 import scipy.misc
 import odl
-import odl.contrib.fom
-from odl.contrib.fom.util import filter_image_sep2d
-from odl.util.testutils import simple_fixture
+from odl.contrib import fom
+from odl.util.testutils import simple_fixture, noise_element
 
 fft_impl = simple_fixture('fft_impl',
                           [odl.util.testutils.never_skip('numpy'),
                            odl.util.testutils.skip_if_no_pyfftw('pyfftw')])
+
 space = simple_fixture('space',
                        [odl.rn(3),
-                        odl.uniform_discr(0, 1, 10)])
+                        odl.rn(3, dtype='float32'),
+                        odl.uniform_discr(0, 1, 10),
+                        odl.uniform_discr([0, 0], [1, 1], [5, 5])])
+
+scalar_fom = simple_fixture('scalar_fom',
```

Comment:
Lots of other FOMS are scalars (e.g. PSNR), is there some other feature that's important to these? Name should reflect this